### PR TITLE
add devdb as an option for import-tool

### PR DESCRIPTION
**Pull request by sheridancbio**

### CODE REVIEW

File: import-scripts/import-devdb-data.sh [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    IMPORTER_JAR_FILENAME="$PORTAL_HOME/lib/devdb-importer.jar"
+    java_debug_args=""
+    ENABLE_DEBUGGING=0
+    if [ $ENABLE_DEBUGGING != "0" ] ; then
+        java_debug_args="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=27183"
+    fi
+    JAVA_IMPORTER_ARGS="$JAVA_PROXY_ARGS $java_debug_args $JAVA_SSL_ARGS -Dspring.profiles.active=dbcp -Djava.io.tmpdir=$tmp -ea -cp $IMPORTER_JAR_FILENAME org.mskcc.cbio.importer.Admin"
+    devdb_notification_file=$(mktemp $tmp/devdb-portal-update-notification.$now.XXXXXX)
+    ONCOTREE_VERSION_TO_USE=oncotree_candidate_release
+    DATA_SOURCES_TO_BE_FETCHED="datahub impact"
```
Comment (averyniceday) [inline import-scripts/import-devdb-data.sh:53]: will need to include whatever new forks we're using to mimic private repos - I can add this once I iron this out with Anusha/Ramya

**Review by averyniceday — CHANGES_REQUESTED** [review]
small request to add the new vars into automation environment - everything else looks good to me :)

File: import-scripts/import-devdb-data.sh [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+FLOCK_FILEPATH="/data/portal-cron/cron-lock/import-devdb-data.lock"
+(
+    # check lock so that script executions do not overlap
+    if ! flock --nonblock --exclusive $flock_fd ; then
+        exit 0
+    fi
+
+    # set necessary env variables with automation-environment.sh
+    if [ -z "$PORTAL_HOME" ] || [ -z "$START_DEVDB_IMPORT_TRIGGER_FILENAME" ] || [ -z "$KILL_DEVDB_IMPORT_TRIGGER_FILENAME" ] || [ -z "$DEVDB_IMPORT_IN_PROGRESS_FILENAME" ] || [ -z "$DEVDB_IMPORT_KILLING_FILENAME" ] ; then
```
[the remark this answers is not in the document]
Comment (sheridancbio) [reply import-scripts/import-devdb-data.sh:11]: good catch - done. Through this I also noticed that the kill script was missing .. so I added that too.

File: import-scripts/import-devdb-data.sh (hunk 1 above)
Comment (sheridancbio) [reply import-scripts/import-devdb-data.sh:53]: 👍  There are some notes in our scrum channel about other things which might need to be tweaked (jar filename, log filename, importer configuration sheet header)

**Comment by sheridancbio** [comment]
@averyniceday I have changed the source data repository back to /data/portal-cron/hgnc-portal-data (to be used for both hgnc imports and also devdb imports). Curators will need to avoid overlapping imports into hgnc and into devdb if the same source repository/fork is used for both of these import processes.

**Review by averyniceday — APPROVED** [review]

**Pull request merged**